### PR TITLE
Fix/map 146 unified snackbar

### PR DESCRIPTION
**Pull request by obozhko-dev**
ExtendSnackbar component added to ForgotPassword, Login, PointPopup, ProfilePage, Registration components.
ExtendSnackbar has default props , severity by default has property 'error', if you need another severity you need sent it as a prop

### CODE REVIEW

File: frontend/src/components/Registration/Registration.tsx [hunk 1]
```diff
@@ -99,35 +98,17 @@ function Registration() {
...
                   {t('registration.createProfile')}
                 </Typography>
-                <Snackbar
-                  anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-                  sx={{ zIndex: 10000 }}
+                <ExtendSnackbar
                   open={!!notification}
-                  autoHideDuration={3000}
+                  notification={notification}
                   onClose={handleCloseNotification}
```
Comment (naberalgamma) [inline frontend/src/components/Registration/Registration.tsx:104]: after successful registration 
image
we see empty snackbar

File: frontend/src/components/Profile/ProfilePage.tsx [hunk 2]
```diff
@@ -119,28 +113,17 @@ export default function ProfilePage() {
...
-          </Snackbar>
-          <Snackbar
-            anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-            sx={{ zIndex: 10000 }}
+            severity="success"
+          />
+          <ExtendSnackbar
             open={!!errorMessage}
+            notification={!!errorMessage}
             onClose={handleClose}
```
Comment (naberalgamma) [inline frontend/src/components/Profile/ProfilePage.tsx:125]: after successful data changing we don't see successful snackbar

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx [hunk 3]
```diff
@@ -79,21 +78,12 @@ function ForgotPassword() {
...
   if (notification) {
     snackbar = (
-      <Snackbar
-        anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-        sx={{ zIndex: 10000 }}
+      <ExtendSnackbar
         open={!!notification}
-        autoHideDuration={3000}
+        notification={notification}
         onClose={handleCloseNotification}
```
Comment (naberalgamma) [inline frontend/src/components/ForgotPassword/ForgotPassword.tsx]: this page crashes after entering email and click to send password, maybe it's not your fault

File: frontend/src/components/Registration/Registration.tsx (hunk 1 above)
Comment (obozhko-dev) [reply frontend/src/components/Registration/Registration.tsx:104]: fixed

File: frontend/src/components/Profile/ProfilePage.tsx (hunk 2 above)
Comment (ChikiBum) [reply frontend/src/components/Profile/ProfilePage.tsx:125]: not fixed, sorry :)

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx (hunk 3 above)
Comment (obozhko-dev) [reply frontend/src/components/ForgotPassword/ForgotPassword.tsx]: fixed

File: frontend/src/components/Profile/ProfilePage.tsx (hunk 2 above)
Comment (6mahdapihka9) [reply frontend/src/components/Profile/ProfilePage.tsx:125]: this will be refixed in the next PR, we can merge this one for now

**Review by 6mahdapihka9 — APPROVED** [review]

**Review by dobbi-free — APPROVED** [review]

**Pull request merged**